.github/workflows/run.sh

```diff
@@ -4,12 +4,16 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+test -z ${FATS_DIR-} && FATS_DIR=`dirname "${BASH_SOURCE[0]}"`/../..
 source ${FATS_DIR}/.configure.sh
 
 # install tools
 ${FATS_DIR}/install.sh riff
 ${FATS_DIR}/install.sh kubectl
 
+NAMESPACE=${NAMESPACE-fats}
```

Comment:
rm

this script should never be invoked by anything other than the CI workflow.